services/worker-manager/README.md

```diff
@@ -136,6 +136,25 @@ end
   updateExpires -.-> cleanupProviders[Cleanup providers]
 ```
 
+#### Sequence of calls
```

Comment:
https://github.com/taskcluster/taskcluster/blob/e8fde4f09e0f6a4f246556fa6d4a03abdf735f68/services/worker-manager/README.md#sequence-of-calls